db/migrate/20160719131003_create_recipes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRecipes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :recipes do |t|
+      t.string :recipe_name
+      t.string :ingredient_list, array: true, default: []
```

Comment:
Nooo you need relation 1 to many. No array field. Just in ingredient table you need field reference